src/Controller/RequestsController.php

```diff
@@ -36,6 +36,8 @@ public function beforeFilter(Event $event)
         if (!Configure::read('debug')) {
             throw new NotFoundException();
         }
+
+        $this->response->header(['Content-Security-Policy' => "default-src 'self'; style-src 'self' fonts.googleapis.com; font-src 'self' fonts.gstatic.com"]);
```

Comment:
Removed or replaced with ```null```? So it will overwrite possible header options which were set before somewhere.